cpp/betweenness_centrality_module/algorithm/betweenness_centrality.cpp

```diff
@@ -81,10 +85,15 @@ std::vector<double> BetweennessCentrality(const mg_graph::GraphView<> &graph, bo
       }
 
       if (current_node != node_id) {
-        if (directed) betweenness_centrality[current_node] += dependency[current_node];
+        if (directed){
```

Comment:
```suggestion
        if (directed) {
```
We have `.clang-format` in mage.